### PR TITLE
Crowdin: Fix and expose Mesh WB translations

### PR DESCRIPTION
**Pull request by luzpaz**
Closes https://github.com/FreeCAD/FreeCAD-translations/issues/54

### CODE REVIEW

**Comment by luzpaz** [comment]
Lol, i can't find the checkbox to set this PR as a Draft. There are still some pending issues that required fixing before merge

**Comment by luzpaz** [comment]
@chennes 
* Who's name shall I add to here? (git blame doesn't tell me who created this file, AFAICT) https://github.com/FreeCAD/FreeCAD/blob/2650baf1e6ffb826eb24539af2c0520b101a4f60/src/Mod/MeshPart/Gui/MeshFlatteningCommand.py#L2
* How can I add translations here? Shall I use the Draft workbench workaround?

**Comment by chennes** [comment]
It looks to me like it was added by @looooo about four years ago. And I don't think we need to do any tricks here, I think (but am not 100% sure) that you can just add calls to `QtGui.QApplication.translate` on the couple of lines that need it. We might have to modify the crowdin script again to scan python files in that directory, I bet it's only scanning C++ right now.

**Comment by yorikvanhavre** [comment]
Is there anything else you need here @luzpaz ? This PR seems OK to me. With @chennes 's changes in the crowdin script, I guess this py file should now be parsed correctly..

**Comment by luzpaz** [comment]
sorry about delays on my part, tending to IRL things. Will take a look at this soon but if anyone wants to take the reigns on some of the open PRs, please do so!

**Comment by berndhahnebach** [comment]
would you rebase the PR to make it run on CI?

**Comment by berndhahnebach** [comment]
Following a link to the branch on the CI-repository:

https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5060

The CI-status is available on the latest commit of the branch.
If there is no status available the PR should be rebased on latest master.
Check pipeline branches to see if your PR has been run by the CI.

https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines?scope=branches

**Comment by luzpaz** [comment]
image

So confused :confused:

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5028">pipeline status</a> for feature branch [PR_5028](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5028). Pipeline [406327264](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/406327264) was triggered at [ab1eab9](https://github.com/FreeCAD/FreeCAD/pull/5028/commits/ab1eab9ea317a2d35866bc31afe745c809a347a0). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by looooo** [comment]
CreateFlatFace can be used to unwrap a face. This has to be one face. Currently there is no support to unwrap multiple faces.
CreateFlatMesh is used to unwrap mesh-surfaces.

Sry for my late response. If I ever have time to work on this topic again I would like to extend the functionality.

**Comment by yorikvanhavre** [comment]
> So confused
I think that's because I fixed a file conflict...

File: src/Mod/MeshPart/Gui/MeshFlatteningCommand.py [hunk 1]
```diff
@@ -69,10 +73,12 @@ def IsActive(self):
...
     """create a flat face from a single face
        only full faces are supported right now"""
-       
+
     def GetResources(self):
-        return {'Pixmap': 'MeshPart_CreateFlatFace.svg', 'MenuText': 'Unwrap Face', 'ToolTip': 'find a flat representation of a mesh'}
-    
+        return {'Pixmap': 'MeshPart_CreateFlatFace.svg',
+                'MenuText': QT_TRANSLATE_NOOP("MeshPart_FlatteningCommand", "Unwrap Face"),
+                'ToolTip': QT_TRANSLATE_NOOP("MeshPart_FlatteningCommand", "Find a flat representation of a mesh.")}
```
Comment (luzpaz) [inline src/Mod/MeshPart/Gui/MeshFlatteningCommand.py:80]: @yorikvanhavre, based on what loooo wrote (https://github.com/FreeCAD/FreeCAD/pull/5028#issuecomment-966509235), I think we need to change the description from `mesh` to `face`

**Pull request merged**